File: code/skill-list-get.py
import autotab
from autotab.rest import ApiException


client = autotab.Client(
    autotab.Configuration(
        api_key = os.environ["AUTOTAB_API_KEY"]
    )
)

try:
    skills = await autotab.SkillApi(client).list()
    print(f"skills: {"\n".join([skill.model_dump_json(indent=2) for skill in skills])}")
except ApiException as e:
    print(f"Exception: {e})

File: code/skill-list-get.ts
import { Configuration, SkillApi } from 'autotab';

const skillClient = new SkillApi(new Configuration({
    apiKey: process.env['AUTOTAB_API_KEY'],
}));

async function main() {
  const skills = await skillClient.list();
  console.log("skills:", skills);
}

main();